experiments/correlation/check_corr_offset.py

```
@@ -0,0 +1,51 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from psfmodel.gaussian import GaussianPSF
+
+from nav.support.correlate import navigate_with_pyramid_kpeaks
+
+def try_corr_offset(image, model, mask, **kwargs):
```

Ruff (0.14.3)

Missing type annotation for `**kwargs`

(ANN003)